src/niamoto/core/services/transformer.py

```
@@ -73,89 +80,246 @@
+            results[group_by_name]["widgets_generated"] = widgets_generated
+            results[group_by_name]["end_time"] = (
+                datetime.now() if hasattr(progress_manager, "_start_time") else None
+            )
+
+            progress_manager.complete_task(
+                task_name, f"{group_by_name} transformation completed"
+            )
+
+        return results
+
+    def _process_configs_simple(self, configs, csv_file, recreate_table):
```

Pylint (3.3.7)

[refactor] Too many local variables (20/15)

(R0914)